src/constants/criteria/criteria-presentation.js

```diff
@@ -1,8 +1,12 @@
+import labelsCriteria from './labels';
```

Comment:
You don't need to import the whole file you can do this 
import { firstImpressionLabel, platingLabel } from './labels';

Reply:
Same for the other two files